ui/index.d.ts

```diff
@@ -473,8 +476,8 @@ export declare abstract class AzureNameStep<T extends IRelatedNameWizardContext>
 export interface IResourceGroupWizardContext extends ILocationWizardContext, IRelatedNameWizardContext {
     /**
      * The resource group to use for new resources.
-     * If an existing resource group is picked, this value will be defined after `ResourceGroupStep.prompt` occurs
-     * If a new resource group is picked, this value will be defined after `ResourceGroupStep.execute` occurs
+     * If an existing resource group is picked, this value will be defined after `ResourceGroupListStep.prompt` occurs
+     * If a new resource group is picked, this value will be defined after the `execute` phase of the 'create' subwizard
      */
     resourceGroup?: ResourceGroup;
```

Comment:
Ideally these related entries would be inside an object e.g.:

   resourceGroup: {
      existing?: ResourceGroup;
      newName?: string;
   }

Reply:
I agree grouping the values makes more sense, but I also think there's more improvements to be made around the wizardContext. Filed an issue to address separately: https://github.com/Microsoft/vscode-azuretools/issues/159